registration/api/v2/views.py

```diff
@@ -292,6 +292,10 @@ def image(self, request, pk: str):
                 application.logo = request.data['logo']
                 application.save()
 
+            if 'hero_cover_image' in request.data:
```

Comment:
Atm, this endpoint only allows uploading and saving an image and doesn't have a way to delete an existing image and reset the value of the `ImageField` to `None`. I will implement the changes for it soon.